Commands/Modules/VideoUtils.cs

```diff
@@ -69,7 +70,7 @@ public class VideoUtils : InteractionModuleBase
 		if (!Directory.Exists("./tmp"))
 		{
 			Directory.CreateDirectory("./tmp");
-			await Program.LogAsync("VideoUtils", "TMP directory not found. Created it automatically.", LogSeverity.Info);
+			Log.Information("TMP directory not found. Created it automatically, VideoUtils");
```

Comment:
Same thing as in `InteractionServiceHandler.cs` basically. Especially since this is normally a one time message, and it's only present in one class.

```suggestion
			Log.Information("TMP directory not found. Created it automatically");
```